src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -259,7 +259,7 @@ def load_arguments(self, _):
         c.argument('target_folder', options_list=['--target-folder'], help='Destination folder to which content will be restored. To restore content to root , leave the folder name empty')
         c.argument('target_storage_account', options_list=['--target-storage-account'], help='Destination storage account to which content will be restored')
         c.argument('source_file_type', arg_type=get_enum_type(['File', 'Directory']), options_list=['--source-file-type'], help='Specify the source file type to be selected')
-        c.argument('source_file_path', options_list=['--source-file-path'], help="""The absolute path of the file, to be restored within the file share, as a string. This path is the same path used in the 'az storage file download' or 'az storage file show' CLI commands.""")
```

Comment:
>source_file_path [](start = 20, length = 16)

please add sample in help.py

Reply:
please also add test case for multi file support

---
In reply to: [390721217](https://github.com/Azure/azure-cli/pull/12522#discussion_r390721217) [](ancestors = 390721217)

Reply:
Added